packages/www/components/AssetDetails/AssetOverviewTab.tsx

```diff
@@ -134,7 +134,9 @@ const AssetOverviewTab = ({
             may also archive them using the following storage providers.
           </Text>
         </Box>
-        <Promo size="2" css={{ borderRadius: "$2" }}>
+        <div
+          className="bg-background text-foreground p-4 rounded-lg border"
+          css={{ borderRadius: "$2" }}>
```

Comment:
When we move to tailwind, can we remove all of the previous CSS? In this, they are competing for priority (they are both applying border radius here).

Reply:
Yep, but we will create this component from scratch with shadcn, so it doesn't make sense to style all of it here again